src/Kentico.Xperience.Lucene.Admin/Kentico.Xperience.Lucene.Admin.csproj

```diff
@@ -11,10 +11,6 @@
     <AdminOrgName>kentico</AdminOrgName>
     <RootNamespace>Kentico.Xperience.Lucene.Admin</RootNamespace>
     <NoWarn>$(NoWarn);S1075</NoWarn>
-    <LangVersion>latest</LangVersion>
```

Comment:
We shouldn't use a different language version than the one available in the SDK version in `global.json`.